getting-started/docs/testing-pays-features.html

```diff
@@ -0,0 +1,190 @@
+---
+layout: getting-started
+active: "testing-pays-features"
+wnt: "Find out some of the things you can do with Testing Pays."
+wnh: "./what-can-i-do"
+wnlt: "What can I do in Testing Pays?"
+---
+
+<h1 class="page-heading">Testing Pays Features</h1>
+
+<div class="docs-text-block">
+  <p>
+    Your Testing Pays account provides features for you to test all aspects of your payments integration. You can concentrate on building the best payment features for your customers.
+  </p>
+
+  <p>
+    Check out some of the features:
+  </p>
+
+  <ul class="doc-list">
+    <li>
+      <a class="doc-link" href="./testing-pays-features#section-api-simulators">
+        API Simulators
+      </a>
+    </li>
+    <li>
+      <a class="doc-link" href="./testing-pays-features#section-response-maps-and-triggers">
+        API Response Maps & Triggers
+      </a>
+    </li>
+    <li>
+      <a class="doc-link" href="./testing-pays-features#section-integration-guides">
+        Integration Guides & Samples
+      </a>
+    </li>
+    <li>
+      <a class="doc-link" href="./testing-pays-features#section-test-reports">
+        Test Reports
+      </a>
+    </li>
+    <li>
+      <a class="doc-link" href="./testing-pays-features#section-live-logs">
+        Live Logs
+      </a>
+    </li>
+    <li>
+      <a class="doc-link" href="./testing-pays-features#section-load-test-simulators">
+        Load Test Simulators
+      </a>
+    </li>
+  </ul>
+</div>
+
+{% include docs/components/text-block-divider.html %}
+
+<div class="docs-text-block" id="section-api-simulators">
+  <h2 class="section-heading">API Simulators</h2>
+
+  <p>
+    Testing Pays has powerful simulators for <i>many</i> API endpoints and operations. For example, there's a simulator for Stripe charges, a simulator for Realex 3D secure flows, a simulator for PayPal Express, etc.
+  </p>
+
+  <p>
+    With a simple copy&paste, you can point your development and test environments at your team's account, then you're ready to send test transactions to the simulators - they'll respond as if your code is talking to the real thing.
```

Comment:
We should use the html encoded version of the amp character

```suggestion
    With a simple copy&amp;paste, you can point your development and test environments at your team's account, then you're ready to send test transactions to the simulators - they'll respond as if your code is talking to the real thing.
```